src/test/java/org/jenkinsci/plugins/workflow/log/LogStorageTestBase.java

```diff
@@ -330,6 +343,13 @@ private AnnotatedLargeText<?> text(String id) {
         return createStorage().stepLog(new MockNode(id), true);
     }
 
+    /**
+     * Concatenate the given lines after interspersing system-dependent line separators between them and adding a final line separator.
+     */
+    private String lines(CharSequence... lines) {
```

Comment:
Maybe [unlines](https://hackage.haskell.org/package/base-4.17.0.0/docs/Prelude.html#v:unlines) or something else would be a better name.

Reply:
`lines` seems intuitive as is.